apps/user-service/Dockerfile

```
@@ -0,0 +1,30 @@
+FROM node:20-alpine AS base
+
+ARG DB_CLOUD_URI
+ARG JWT_SECRET
+ENV DB_CLOUD_URI=${DB_CLOUD_URI}
+ENV JWT_SECRET=${JWT_SECRET}
+# ARG NODE_ENV=development
+# ENV NODE_ENV=${NODE_ENV}
+ENV ENV=PROD
+
+FROM base as deps
```

Should change this to avoid the `FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 11)` warning

```suggestion
FROM base AS deps
```